### ISSUE TITLE
Recursive aggregation in meta-aggregate Yes or No? Implement if yes

### ISSUE THREAD

**Issue by christian-monch** [datalad/datalad-metalad#101]
The current implementation of meta-aggregate does not support recursive aggregation:

We have to decide whether we want to implement recursion in `meta-aggregate` or leave
it to a higher level logic.

In case of a positive decision, the implementation has to be performed.

(The semantics described in an earlier version of the code:

    Metadata aggregation can be performed recursively, in order to aggregate
    all metadata from all subdatasets. By default, re-aggregation of metadata
    inspects modifications of datasets and metadata extractor parameterization
    with respect to the last aggregated state. For performance reasons,
    re-aggregation will be automatically skipped, if no relevant change is
    detected. This default behavior can be altered via the ``--force``
    argument.
)

**Comment by mih** [comment]
I agree. Regarding the API design, this might be an aspect of a more general issue. The functionality grows, starting from simple building blocks. Maybe we should use more specific command names and leave the more general slots in the namespace for later high-level commands.

Nothing to decide for now, but to be considered before going for a release.

**Comment by christian-monch** [comment]
I agree, This is related to a general "design"-idea that was outlined in issue #68 here https://github.com/datalad/datalad-metalad/issues/68#issuecomment-797352569.

(mainly: keep higher level logic in higher-level "orchestration"-modules)
